src/wp-includes/class-wp-object-cache.php

```diff
@@ -323,6 +323,67 @@ public function get_multiple( $keys, $group = 'default', $force = false ) {
 		return $values;
 	}
 
+	/**
+	 * Delete multiple values from the cache in one call.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param array  $keys  Array of keys to be delete.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @return array Array of return values organized into groups.
+	 */
+	public function delete_multiple( $keys, $group = '' ) {
+		$values = array();
+
+		foreach ( $keys as $key ) {
+			$values[ $key ] = $this->delete( $key, $group );
+		}
+
+		return $values;
+	}
+
+	/**
+	 * Add multiple values from the cache in one call.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param array  $data  Array of key and value to be added.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @param int    $expire Optional. When to expire the cache contents, in seconds.
+	 *                           Default 0 (no expiration).
+	 * @return array Array of return values organized into groups.
+	 */
+	public function add_multiple( $data, $group = '', $expire = 0 ) {
```

Comment:
Same as above as well.

```suggestion
	public function add_multiple( array $data, $group = '', $expire = 0 ) {
```

Reply:
Actioned in https://github.com/WordPress/wordpress-develop/pull/2018